README.md

```
@@ -3068,7 +3068,19 @@ Bug reports must include:
 </details>
 
 <details>
-<summary><code>#241</code> <strong>Universal</strong> &middot; 38 patches</summary>
+<summary><code>#241</code> <strong>Zutobi</strong> &middot; 1 patch &middot; <code>com.driverlicenseapp</code> &middot; <a href="https://play.google.com/store/apps/details?id=com.driverlicenseapp">Play Store</a></summary>
+<br>
+
+**Supported versions:** `4.5.78`
+
+| Patch | Details |
+|---|---|
+| [**Unlock Premium**](#unlock-premium) | Unlocks Driver's Ed premium features by patching the Flutter subscription gate. |
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)